docs/index.md

```diff
@@ -0,0 +1,125 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The Payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK allows for client-side approval of an order with the user's card information.
+
+## Eligibility
+```
+# Placeholder: Product Eligibility
+what kind of merchants are we targetting?
+what features do we offer right now/what features are we missing?
+```
+
+## Integration methods
+
+- [Card Fields UI Integration](#card-fields-ui-integration)
+- [Fully Customizable Integration](#fully-customizable-integration)
+
+## Card Fields UI Integration
+
+Accept card payments using our card fields UI
+
+```
+# Placeholder: Include images of card fields
+```
+
+**Requirements:**
+Card fields UI offers a low effort integration where you can use our card fields UI, and our card fields will handle ...
+
+## Fully Customizable Integration
+
+Process card payments with PayPal Payments SDK using your own UI.
+
+### Steps
+
+#### 1. Create an order
+Call `v2/checkout/orders` to create an order an obtain an order ID
+
+> NOTE: field values enclosed in angle brackets '<>' are placeholders
+
+```
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <Access-Token>' \
+--data-raw '{
+    "intent": "<CAPTURE|AUTHORIZE>",
+    "purchase_units": [
+        {
+            "amount": {
+                "currency_code": "USD",
+                "value": "5.00"
+            }
+        }
+    ]
+}'
+```
+
+##### Additional Links
+
+- [PayPal Developer: Get Started – Get an access token](https://developer.paypal.com/api/rest/#link-getanaccesstoken)
+- [PayPal Developer: Orders – Create an order](https://developer.paypal.com/api/orders/v2/#orders_createe)
```

Comment:
This URL works but looks like `#orders_create` also points to the same thing 😄

Reply:
Yeah that's how the link icon copied it lol I saw that too. I can remove the extra e.